js/ui/list/ui.list.base.js

```diff
@@ -978,6 +1017,7 @@ var ListBase = CollectionWidget.inherit({
         this._createComponent($button, Button, {
             text: this.option("nextButtonText"),
             onClick: this._nextButtonHandler.bind(this),
+            type: /material/.test(themes.current()) ? "default" : undefined,
```

Comment:
extract a method `isMaterialTheme`

Reply:
We decided to do it afterward. Having a method isn't the good idea as well - it's better to have a global constant.

Reply:
ok, global constant is good. It is necessary to use a regular expression in a single place